CHANGELOG.md

```diff
@@ -851,6 +851,7 @@
 - [Added retries when executing GraalVM updater][7079]
 - [Add method call info for infix operators][7090]
 - [`executionComplete` response is sent on successful execution only][7143]
+- [Update to GraalVM 23.0.0][7176]
 - [Send info about function values][7168]
```

Comment:
Why not in ascending order?

Reply:
Probably the default behavior of `git merge develop`. Reverted in 611212cf69922e0891863ac73f11da50ae1c0c27